include/xeus-sqlite/xeus_sqlite_interpreter.hpp

```diff
@@ -176,6 +171,10 @@ class XEUS_SQLITE_API interpreter : public xeus::xinterpreter
     std::string m_db_path;
 };
 
+    /**
+     * Cleans the code from inputs that are acceptable in a jupyter notebook.
+     */
+    XEUS_SQLITE_API std::string sanitize_string(const std::string& code);
```

Comment:
Maybe this should be put under a `detail` namespace, as it is not really public user API

Reply:
See this for example https://github.com/jupyter-xeus/xeus-python/blob/ec7338cafbe0f172bf9f1dcd0c0088713b35d167/src/xcomm.cpp#L164

Reply:
Right, thanks for the link.
I'll do this once Travis decide the code is good.
https://travis-ci.com/github/jupyter-xeus/xeus-sqlite/jobs/319214854
Or... Is this problem caused by it somehow?

Reply:
No this was just a network issue. I restarted the build. Let's see